custom_components/smartthinq_sensors/wideq/devices/ac.py

```diff
@@ -690,6 +690,8 @@ async def set_fan_speed(self, speed):
         """Set the fan speed to a value from the `ACFanSpeed` enum."""
         if speed not in self.fan_speeds:
```

Comment:
```suggestion
        if speed.lower() not in self.fan_speeds:
```

Reply:
Probably is better to also put here a lower() to not break existing configured service calls